test/cqlpy/test_custom_index.py

```diff
@@ -0,0 +1,75 @@
+# Copyright 2025-present ScyllaDB
+#
+# SPDX-License-Identifier: LicenseRef-ScyllaDB-Source-Available-1.0
+
+###############################################################################
+# Tests for custom indexes
+###############################################################################
+
+import pytest
+from .util import new_test_table
```

Comment:
Please import is_scylla() from .util as well, and you won't need to re-implement it below.